docs/auth/federated-auth.md

```diff
@@ -224,6 +224,32 @@ Future<UserCredential> signInWithApple() async {
 }
 ```
 
+#### Apple platform sign-in only
+
+Apple sign-in on iOS+ platforms can also be achieved with the following method:
+
+```dart
+// Create a nonce
+String rawNonce = createNonce();
```

Comment:
Some guidance on generating it would be nice. At least clarify that the reader needs to implement this.

Reply:
@kevinthecheung - thanks, I've updated the docs 👍